core/store/orm/config.go

```diff
@@ -1168,8 +1178,20 @@ func (c Config) MinRequiredOutgoingConfirmations() uint64 {
 // supplied for a contract to be considered.
 func (c Config) MinimumContractPayment() *assets.Link {
 	minimumContractPayment := chainSpecificConfig(c).MinimumContractPayment
-	if c.viper.IsSet(EnvVarName("MinimumContractPayment")) || minimumContractPayment == nil {
+	if c.viper.IsSet(EnvVarName("MinimumContractPayment")) {
 		return c.getWithFallback("MinimumContractPayment", parseLink).(*assets.Link)
+
+		// TODO: Remove when implementing
+		// https://app.clubhouse.io/chainlinklabs/story/8096/fully-deprecate-minimum-contract-payment
+	} else if c.viper.IsSet("MINIMUM_CONTRACT_PAYMENT") {
+		str := c.viper.GetString("MINIMUM_CONTRACT_PAYMENT")
+		value, ok := new(assets.Link).SetString(str, 10)
```

Comment:
Doesn't this change the behaviour from what it used to mean?

Reply:
I believe this is doing exactly what it was doing before, i.e. this is just the body of parse parseLink.